src/templates/partials/menu.hbs

```diff
@@ -18,8 +18,8 @@
 
     <h2>Hannah Sutton Design</h2>
     <p>Melbourne, Australia</p>
-    <a href="#">create@hannahsuttondesign.com</a>
-    <a href="#">m +61 (0) 428 226 905</a>
+    <a href="mailto:create@hannahsuttondesign.com">create@hannahsuttondesign.com</a>
```

Comment:
Ideally always include `target="_blank"` so the email opens into a different screen. The last thing you want is for Hannah's website to be replaced with Gmail inbox or whatever default mail client they have.